src/pages/Account/AccountSecurity.tsx

```diff
@@ -1,10 +1,72 @@
 import * as React from 'react';
+import { ListGroup, ListGroupItem, Alert } from 'reactstrap';
+import { resetPassword } from 'utils/firebase';
+import { getDisplaySuccessMessage, SuccessMessage, getDisplayErrorMessage, ErrorMessage } from 'utils/validators';
+import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+import { User } from 'firebase';
 
-class AccountSecurity extends React.Component<any> {
-  render() {
-    return (
-      <h1>This is the Security Page</h1>
-    );
+interface AlertProperties {
+  color: string;
+  msg: string;
+  show: boolean;
+}
+
+function AccountSecurity() {
+  const [alert, setAlert] = React.useState<AlertProperties>({ color: '', msg: '', show: false });
+  const [isSubmitting, setSubmitting] = React.useState<boolean>(false);
+
+  return (
+    <>
+      <Alert
+        isOpen={alert.show}
+        color={alert.color}
+        toggle={() => setAlert({ ...alert, show: false })}>
+        {alert.msg}
+      </Alert>
+
+      <FirebaseConsumer>
+        {({ user }: FirebaseUserProps) => (
+          <ListGroup className="mb-2 d-flex flex-column">
+            {user &&
+              <ListGroupItem
+                className="w-100 d-flex flex-column"
+                disabled={isSubmitting}
+                onClick={() => handleClick(user)}>
```

Comment:
can `handleClick` be renamed to what it does? is it `handleResetPasswordClick`?

Reply:
👍